language/stdlib/modules/LibraConfig.move

```diff
@@ -224,6 +224,12 @@ module LibraConfig {
 
         let config_ref = borrow_global_mut<Configuration>(CoreAddresses::LIBRA_ROOT_ADDRESS());
         let current_time = LibraTimestamp::now_microseconds();
+
+        // Do not do anything if a reconfiguration event is already emitted within this transaction.
```

Comment:
I think it would be worth explaining this in a lot more detail because it's pretty subtle:
- The time changes in every non-empty block
- A block automatically ends after a transaction that emits a reconfiguration event
- Each transaction must emit at most one reconfiguration event
- Thus, this check ensures that a transaction that does multiple "reconfiguration required" actions emits only one reconfiguration event

Reply:
Just added the comments with these notes and a few extra details. Lemme know how they look to you!